src/components/not-found.jsx

```diff
@@ -1,3 +1,18 @@
+/* global CONFIG */
 import React from 'react';
+import { Helmet } from 'react-helmet';
 
-export default () => <div className="commentBox">Nothing here :(</div>;
+export function NotFound() {
+  return (
+    <div className="box">
+      <Helmet title={`Page not found - ${CONFIG.siteTitle}`} defer={false} />
+      <div className="box-header-timeline">Page not found</div>
+      <div className="box-body">
+        <p className="alert alert-danger">
+          There is no page with such address on {CONFIG.siteTitle}.
```

Comment:
There is no page with such address on => 
There is no page with such an address on